src/Dockerfile-dev

```diff
@@ -1,4 +1,4 @@
-FROM pyroapi:python3.8-alpine3.10
+FROM pyronear/pyro-api:python3.8-alpine3.10
```

Comment:
@blenzi your are changing this in severals PR it should be done once in a PR I guess

Reply:
Thanks @MateoLostanlen. Now that #281 was merged, this will be part of main